packages/experiments/src/components/CollapsibleSection/CollapsibleSectionPage.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {
+  ExampleCard,
+  IComponentDemoPageProps,
+  ComponentPage,
+  PropertiesTableSet
+} from '@uifabric/example-app-base';
+
+import { CollapsibleSectionRecursiveExample } from './examples/CollapsibleSection.Recursive.Example';
+const CollapsibleSectionRecursiveExampleCode =
+  /* tslint:disable-next-line:max-line-length */
+  require('!raw-loader!@uifabric/experiments/src/components/CollapsibleSection/examples/CollapsibleSection.Recursive.Example.tsx') as string;
+
+export class CollapsibleSectionPage extends React.Component<IComponentDemoPageProps, {}> {
```

Comment:
I'm in the middle of refactoring demo pages, FYI in #5089

Reply:
As we discussed verbally, this conversion hasn't yet been done for any experiments pages, but will be done on promotion